.github/workflows/PR-4.x.yaml

```diff
@@ -2,6 +2,7 @@ name: PR:4.x
 
 on:
   pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
```

Comment:
> By default, a workflow only runs when a pull_request event's activity type is `opened`, `synchronize`, or `reopened`

I guess we want to re-run jobs, if labels were changed or added by a maintainer.

Reply:
But there is an issue when PR is created with labels. It runs several workflows (`opened` and `labeled` worked). However, sometimes PR are created without any labels.

If to keep the default statement and add labels after some period of time for a PR, need to re-commit, because a workflow won't be triggered in this case.

Reply:
Removed this line accordingly to https://github.com/opencv/opencv/pull/22112#discussion_r899325558.